calcom/cal.com
# feat(i18n): finish pt-br translation

## What does this PR do?

This PR adds PT-BR translations that we're missing (about 20%).

Before:
<img width="1312" alt="Screenshot 2024-05-20 at 23 41 28" src="https://github.com/calcom/cal.com/assets/9114644/fa486464-4798-4a98-8241-da38b6dc4e7e">

After:
<img width="1311" alt="Screenshot 2024-05-20 at 23 39 27" src="https://github.com/calcom/cal.com/assets/9114644/af0b5465-c6d0-472d-8a04-436a791005b9">


## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

- Go to "General: on the sidebar and change language to "Português (Brasil)"
- Go to "Ausente" on the sidebar
- Now you can see an example of the missing translation, where the central message is in English, where it should be in Portuguese

- Are there environment variables that should be set? No
- What are the minimal test data to have? Current version has everything needed
- What is expected (happy path) to have (input and output)? See text in Brazilian Portuguese
- Any other important info that could help to test that PR

## Checklist
